gdb/parse.c

```diff
@@ -930,6 +930,7 @@ operator_length_standard (const struct expression *expr, int endpos,
     case BINOP_VAL:
     case UNOP_CAST:
     case UNOP_MEMVAL:
+    case STRUCTOP_ANONYMOUS:
```

Comment:
Oh, btw, historically at least these dumping changes didn't need tests, so I wouldn't bother trying to write one.